mailing.go

```diff
@@ -83,45 +78,41 @@ func (r *Request) ParseTemplate(templateFileName string, data interface{}) error
 	return nil
 }
 
-func (srv *mailingAPI) SendEmails(ctx context.Context, in *mailingv1.SendEmailsRequest) (*mailingv1.SendEmailsResponse, error) {
+func (srv *mailingAPI) SendEmails(ctx context.Context, req *SendEmailsRequest) error {
```

Comment:
Suggestion : `CreateEmailsRequests` / `PrepareEmailRequests` / `FormatEmails` / `PostEmails`.
Everything excepted `SendEmails` because it doesn't send it's formating and looking for the adresses